core/monitoring/user-events/build.gradle

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'scala'
+apply plugin: 'application'
+apply plugin: 'org.scoverage'
+
+ext.dockerImageName = 'monitoring-user-events'
+apply from: "../../../gradle/docker.gradle"
+distDocker.dependsOn ':common:scala:distDocker', 'distTar'
+
+project.archivesBaseName = "openwhisk-monitoring-user-events"
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile "org.scala-lang:scala-library:${gradle.scala.version}"
+    compile ('com.github.pureconfig:pureconfig_2.12:0.9.2') {
+        exclude group: 'com.github.pureconfig', module: 'pureconfig-macros_2.12'
+    }
+    compile 'io.spray:spray-json_2.12:1.3.4'
+
+    compile 'com.typesafe.akka:akka-actor_2.12:2.5.17'
```

Comment:
We can now add dependency to `project(':common:scala')` and remove explicit dependency on akka and various other libraries from here